packages/react/src/components/DropBox/DropBox.js

```diff
@@ -1,21 +0,0 @@
-import React from 'react';
-import PropTypes from 'prop-types';
-import { dropBoxStyles as styles } from './DropBox.styles';
-import { Box } from '../Box';
-
-const DropBox = ({ fullScreen = false, height, children }) => (
```

Comment:
this was never used?

Reply:
Dropbox had a lot of bugs, causing issues while dragging and dropping. So I have removed it for the time being. The drag and drop hooks already handle it well. For the UI of drag and drop, maybe we'll introduce a new component later. This removal was necessary to ensure the project remains bug-free. So yep, currently it is not being used anywhere.

Reply:
got it